MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include versioneer.py
 include nvflare/_version.py
+include nvflare/libs/*.so
```

Comment:
The code also supports .dylib and .dll. Should we add those too for future proof?

Reply:
Only Linux now.  Once we support other platform, we can add more.